resources/js/router/index.ts

```diff
@@ -1,12 +1,15 @@
 import { createRouter, createWebHistory, RouteRecordRaw } from 'vue-router'
 import About from '../pages/About.vue'
 import Home from '../pages/Home.vue'
-import Register from '../pages/auth/Register.vue'
+import Facilities from '../pages/Facilities.vue'
+import Form from '../components/facilities/Form.vue'
 
 const routes: RouteRecordRaw[] = [
     { path: '/', component: Home },
     { path: '/about', component: About },
-    { path: '/register', component: Register }
+    { path: '/facilities', component: Facilities, children: [
+        { path: '/facilities/create', component: Form }
```

Comment:
I think this is wrong...
Shouldn't it be `{ path: '/create', component: Form }` ???